azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -694,6 +699,12 @@ private void doRun() {
         writeStatus();
         fireEvent(Event.create(this, EventType.JOB_STATUS_CHANGED,
             new EventData(Status.RUNNING, this.node.getNestedId())));
+        try {
+          addGroupAndUser(this.jobtypeManager.getCommonPluginLoadProps());
+        } catch (Exception e) {
+          finalStatus = changeStatus(Status.FAILED);
+          logError("Job run failed while adding group and user.");
```

Comment:
just curious: why is this not considered and infrastructure failure and therefore the status picked was FAILED instead of EXECUTION_STOPPED.

Reply:
This is Job status. So that should be marked as Failed. Isn't it?